com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/TestUINodes.cs

```diff
@@ -101,4 +102,36 @@ internal class TestUIVec4Node : IStandardNode
             new ParameterDescriptor("Out", TYPE.Vec4, GraphType.Usage.Out)
         );
     }
+
+    internal class TestUIColorRGBNode : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1, // Version
+            "TestUIColorRGB", // Name
+            "Out = In;",
+            new ParameterDescriptor("In", TYPE.Vec3, GraphType.Usage.Static),
+            new ParameterDescriptor("Out", TYPE.Vec3, GraphType.Usage.Out)
+        );
+
+        public static Dictionary<string, float> UIHints => new()
+        {
+            {"In.UseColor", 0} // Use color picker for In
```

Comment:
I suspect that the value here is ignored but this looks like `In.UseColor` is false because it is 0.

Reply:
I'll make it `1` and add a note that the value is ignored. It does indeed look like a 1/0 boolean value without additional context.